autofill/autofill-impl/src/main/res/values/donottranslate.xml

```diff
@@ -16,4 +16,5 @@
 
 <resources>
     <string name="autofill_service_suggestion_search_passwords">Search Passwords</string>
+    <string name="autofill_service_choose_password_activity">Choose a saved password</string>
```

Comment:
nit: change to `Change a Saved Password`

Reply:
using "Select a Saved Password" after copy review